frontend/vite-project/src/components/shipping-form/ShippingForm.tsx

```diff
@@ -0,0 +1,123 @@
+import { useShippingStore } from "../../stores/shippingStore";
+import './shippingForm.css'
+
+const ShippingForm = () => {
+  const { formData, updateFormField } = useShippingStore();
+
+  return (
+    <section className="section-wrapper">
+      <form
+        className="shipping-form"
+        onSubmit={(e) => {
+          e.preventDefault();
+          console.log("Shipping form submitted", formData);
+        }}
+      >
+        <div className="input-container-flex-col">
+          <p className="section-title">Shipping Information</p>
+
+          <div className="input-group">
+              <div className="label-input">
+                <label htmlFor="name">Name</label>
```

Comment:
Since we don't have labels in the wireframe, in my component I added all necessary labels but, using a class from tailwind called `sr-only` we can make the label visually hidden but screen reader accessible.

```suggestion
                <label htmlFor="name" className="sr-only">Name</label>
```